Capstone_Project/frontEnd/src/components/Home/Homepage.jsx

```diff
@@ -0,0 +1,167 @@
+                                        course.map(({ course_Name }) => 
+                                        <div className="leftborder">
+                                                <h3 className='course' onClick={()=>clickMe()}>
+                                                    {course_Name}
+                                                </h3>
+                                        </div>)
+                                        :""
+                                        //  <><img src="/resources/miccymouse.svg" alt="mouse" className="mouse"/><h1 className="mouse1">WELCOME</h1></>
+                                    }
+                            </div>
+                    <div className="unitshow">
+                        {
+                        isClickMe?
+                        units.map(({ unit_Name}) =>
+                                <div className="unitmargin">
+                                    <div className='units'  onClick={()=>ClickMeOnce()}>
+                                        <div className="rambus">
+                                            <img src="/resources/Rhombus.png" alt="" className="a" />
+                                        </div>
+                                        <div className="text">
+                                            <div className="c" >
+                                                {unit_Name}
+                                                {
+                                                isClicked ? 
+                                                    <div>
+                                                        {
+                                                            subunits.map(({ subUnit_Name,subUnit_Content}) => <div>
+                                                                <div className='subunits' >
+                                                                    <div className="circle" ></div>
+                                                                    <p>
+                                                                        {subUnit_Name}
+                                                                    </p>
+                                                                </div>
+                                                                <p className="content">{subUnit_Content}  </p> 
+                                                                
+                                                            </div>) 
+                                                        }
+                                                    </div> : ""
+                                                }
+                                            </div>
+                                        </div>
+                                        <div className="dropdown">
+                                            <img src="/resources/downarrow.png" alt="" className="b" />
```

Comment:
please add alternate text for image tag.

Reply:
Resolved